### DISCUSSION TITLE
请问Blog的Cover是否有办法平铺？

### DISCUSSION THREAD

**Discussion by EJayyyyyyyyy** [volantis-x/hexo-theme-volantis#730] [Q&A] [answered]
RT,如果拖动窗口就会导致Cover各种形变，十分影响观感，感谢
![image](https://user-images.githubusercontent.com/54256289/160167709-6b9c5a38-c9ae-4064-b173-00a49450365a.png)

**Comment by Tikas** [comment, marked as answer]
这个本身就有多种方式，请查看相关文档再来，谢谢
